src/DependencyInjection/Configuration.php

```diff
@@ -89,8 +87,8 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->children()
                         ->scalarNode('command_bus')
                             ->cannotBeEmpty()
-                            ->defaultValue('message_bus')
-                            ->example('message_bus')
+                            ->defaultValue('sylius_default.bus')
```

Comment:
I would rather we not change this because users who already installed the plugin we know they will have this message bus service if they made the plugin work, but new users will figure it out when they try to install and make it work